EIPS/eip-9698.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 9698
+title: Exponential Gas Limit Increase
+description: Exponentially increase gas limit 100x over 4 years by setting a default for the client vote
+author: Dankrad Feist (@dankrad)
+discussions-to: https://ethereum-magicians.org/t/eip-9698-exponential-gas-limit-increase-via-default-client-voting-behavior/23884
+status: Draft
+type: Informational
+created: 2025-04-27
+---
+
+## Abstract
+
+This proposal introduces a deterministic gas limit growth schedule via client-side defaults. Ethereum clients will vote to increase the gas limit according to an exponential schedule unless explicitly configured otherwise by the user. The gas limit increase occurs every beacon chain epoch, aligned to a factor-of-10 increase every approximately 164,250 epochs (2 years). It will stop after 4 years when an updated gas increase schedule should be decided and committed to.
+
+## Motivation
+
+The current gas limit mechanism relies on miner/operator voting, which lacks coordination and predictability. While flexible, this approach can lead to stagnation or overly cautious increases. By introducing a predictable exponential growth pattern as a client default, this EIP encourages a sustainable and transparent gas limit trajectory, aligned with expected advancements in hardware and protocol efficiency.
+
+## Specification
+
+### Schedule
+
+Let `G0 = 50,000,000` be the gas limit at the activation epoch. Let the activation epoch be Ethereum beacon chain **epoch 369017**, which corresponds to approximately June 1, 2025.
+
+Let `t` be the number of beacon chain epochs since the activation epoch (Epoch 0).
```

Comment:
I think we need a `t` and a `t_0` here where `t_0` is the activation epoch and `t` is the current epoch. Then for mainnet, `t_0` is 369017. Then `G(t) = round(G0 * m^(t-t_0))` and is only valid if `t >= t_0`